dev/docker/hive/Dockerfile

```diff
@@ -181,6 +195,9 @@ RUN chown -R datastrato:hadoop /home/datastrato
 RUN rm -rf /tmp/packages
 RUN rm -rf /var/lib/apt/lists/*
 
+RUN touch ${HIVE_HOME}/conf/orig.txt
+RUN touch ${HIVE_HOME}/lib/orig.txt
```

Comment:
Why do we need to add these codes? is a temporary test code?

Reply:
Sorry, removed.